numpy/core/records.py

```diff
@@ -211,21 +211,25 @@ def _setfieldnames(self, names, titles):
         if _dup:
             raise ValueError("Duplicate field names: %s" % _dup)
 
-        if (titles):
+        if titles:
             self._titles = [n.strip() for n in titles[:self._nfields]]
         else:
             self._titles = []
             titles = []
 
-        if (self._nfields > len(titles)):
+        if self._nfields > len(titles):
             self._titles += [None] * (self._nfields - len(titles))
 
     def _createdescr(self, byteorder):
-        descr = sb.dtype({'names':self._names,
-                          'formats':self._f_formats,
-                          'offsets':self._offsets,
-                          'titles':self._titles})
-        if (byteorder is not None):
+        descr = sb.dtype(
+            {
```

Comment:
If you're going to change this at all, please combine these two lines, and same for the two closing lines.

Reply:
Hello Eric, 
what do you think about running _black_ on the file?

Reply:
I would probably not do that as part of this PR. Let's just change these two lines to the format:
```python
descr = np.dtype({
    'somekey': some_value,
    ...
})
```
Feel free to open a new pr after this one to apply black, but I expect that it will be contentious and may end up sitting unmerged.

Reply:
I understand, I've made the change as requested